### PR TITLE
Update CFCallNumber.m

### PR DESCRIPTION
**Pull request by jufracaqui**
Solved problem in iOS that prevented the app from making phone calls

### CODE REVIEW

**Comment by jufracaqui** [comment]
This references this issue https://github.com/Rohfosho/CordovaCallNumberPlugin/issues/36

File: src/ios/CFCallNumber.m [hunk 1]
```diff
@@ -1,34 +1,24 @@
...
+ 
         CDVPluginResult* pluginResult = nil;
         NSString* number = [command.arguments objectAtIndex:0];
-        number = [number stringByAddingPercentEscapesUsingEncoding:NSUTF8StringEncoding];
-
-        if( ! [number hasPrefix:@"tel:"]){
-            number =  [NSString stringWithFormat:@"tel:%@", number];
-        }
-
-        if(![[UIApplication sharedApplication] canOpenURL:[NSURL URLWithString:number]]) {
```
Comment (nishilshah17) [inline src/ios/CFCallNumber.m]: Why did you remove error handling?

Comment (jufracaqui) [inline src/ios/CFCallNumber.m]: Oops, i forked the repo and pulled to the wrong place, sorry about that. 

The thing is that the plugin stops at this line 
`number = [number stringByAddingPercentEscapesUsingEncoding:NSUTF8StringEncoding];`
and at all the error handling stuff.

I couldn't find the problem myself, i can't programm objective-c, so all i've done is concat "tel:" to the number you get from the javascript and make the phonecall with that.

Comment (nishilshah17) [inline src/ios/CFCallNumber.m]: Yeah, we can't remove error handling in case some apps rely on it; best if you just use your own fork.

Comment (romainbebert) [inline src/ios/CFCallNumber.m]: Is this iOS error being looked at or should removing this error handling part become the workaround if calling from iOS is needed ?
